src/d_deh.c

```diff
@@ -2229,7 +2229,7 @@ void deh_procSounds(DEHFILE *fpin, FILE* fpout, char *line)
                   S_sfx[indexnum].volume = value;
                 else
                   if (!strcasecmp(key,deh_sfxinfo[6]))  // Zero 4
-                    S_sfx[indexnum].data = (void *)(intptr_t) value; // killough 5/3/98: changed cast
+                    ; // ignored
```

Comment:
I think we can also get rid of the `usefulness` field, because it turns out to be, well, useless. 😉

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/971/commits/eda77ffd224b77ad14539f6a9d5601ceebd710ce